core/src/main/codegen/templates/Parser.jj

```diff
@@ -5652,6 +5652,24 @@ SqlCall withinGroup(SqlNode arg) :
     }
 }
 
+SqlCall nullTreatment(SqlCall arg) :
+{
+    final Span span;
+}
+{
+    (
+        <IGNORE> { span = span(); } <NULLS> {
+            return SqlStdOperatorTable.IGNORE_NULLS.createCall(
+                span.end(this), arg);
+        }
+    |
+        <RESPECT> { span = span(); } <NULLS> {
+            return SqlStdOperatorTable.RESPECT_NULLS.createCall(
+                span.end(this), arg);
+        }
+    )
+}
+
```

Comment:
Isn't this part (RESPECT/IGNORE NULLS) already in master by commit: https://github.com/apache/calcite/commit/d7946a94adfd2e788f5d324910944dd65dab11ee ?

Reply:
There are 3 commits in this PR. It looks like the first commit  [0c0e90](https://github.com/apache/calcite/pull/1141/commits/0c0e908b64949095af1c9c21815880089f52c34f) is the very commit.

Reply:
Yes, multiple PRS are here

Reply:
I pushed the first 2 commits to master very soon after opening this PR. The delta between master and this PR is therefore just one commit, 0c0e908b64949095af1c9c21815880089f52c34f. I assumed that github would figure that out, but it didn't. Please just consider that commit.